lectures/networks.md

```diff
@@ -100,7 +100,7 @@ mystnb:
   image:
     alt: aircraft_network
     classes: shadow bg-primary
-    width: 200px
+    width: 500px
```

Comment:
@mmcky It doesn't seem like a good idea for us to choose these manually, since we'll end up with different size figures throughout the document.  Is there a sensible default?

Reply:
These should be transitioned to use `%` such as

````md
```{code-cell} ipython3
---
mystnb:
  figure:
    caption: "2016 US Lorenz curves"
    name: lorenz_us
  image:
    alt: lorenz_us
    classes: shadow bg-primary
    width: 75%
---
fig, ax = plt.subplots()

ax.plot(f_vals_nw[-1], l_vals_nw[-1], label=f'net wealth')
ax.plot(f_vals_ti[-1], l_vals_ti[-1], label=f'total income')
ax.plot(f_vals_li[-1], l_vals_li[-1], label=f'labor income')
ax.plot(f_vals_nw[-1], f_vals_nw[-1], label=f'equality')

ax.legend(fontsize=12)   
plt.show()
```
````